OloEngine/src/OloEngine/Renderer/Commands/CommandMemoryManager.cpp

```
@@ -207,4 +237,136 @@ namespace OloEngine
+
+        // Map thread to worker index
+        {
+            std::scoped_lock<std::mutex> lock(s_WorkerMapMutex);
+            s_ThreadToWorkerIndex[threadId] = workerIndex;
+        }
+
+        CommandAllocator* allocator = GetWorkerAllocator(workerIndex);
+        return { workerIndex, allocator };
+    }
+
+    i32 CommandMemoryManager::GetCurrentWorkerIndex()
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)